package_policies/M16.md

```diff
@@ -1,9 +1,5 @@
-**M16.** The package must support production-quality installation compatible with the xSDK install tool
-and xSDK metapackage.<sup>9</sup>
+**M16** Any xSDK-compatible package that compiles code should have a configuration option to build in **Debug** mode. Debug mode must produce debugging symbols in compiled code and may contain further useful debugging information and additional error checking. Optimization should be disabled in the debug build.
```

Comment:
> Optimization should be disabled in the debug build.

This seems reasonable, but I don't see where this was required in the current installation policies, so it could introduce a new requirement that some packages may not be compliant with. Thus, I think we should strike this line.

Reply:
Ok, I can strike this, and make an issue to discuss. Think it would be good to get consistency.

Reply:
Done